core/templates/domain/exploration/exploration-id-validation.service.ts

```diff
@@ -50,6 +51,24 @@ export class ExplorationIdValidationService {
           return response.correctness_feedback_enabled;
         });
   }
+
+  async isCategoryNotDefault(explorationId: string): Promise<boolean> {
```

Comment:
Maybe better to call this `isDefaultCategoryAsync`?

Reply:
Done!